virt/post_installation_configuration/virt-post-install-storage-config.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="virt-post-install-storage-config"]
+= Post-installation storage configuration
+include::_attributes/common-attributes.adoc[]
+:context: virt-post-install-storage-config
+
+toc::[]
+
+After installing {VirtProductName}, you can configure a default storage class for the Container Storage Interface (CSI) or local storage by using the Hostpath Provisioner (HPP).
+
+[id="defining-default-storage-class"]
+== Defining a default storage class for CSI
+
+You must define a xref:../../post_installation_configuration/storage-configuration.adoc#defining-storage-classes_post-install-storage-configuration[default storage class] for the Container Storage Interface (CSI). Otherwise, the cluster cannot receive automated boot source updates.
+
+[id="configuring-local-storage-hpp"]
+== Configuring local storage by using the HPP
+
+You can configure local storage for virtual machines by using the hostpath provisioner (HPP).
```

Comment:
Yes, you can do this, but there is the real chance of filling up the host path, and borking a bunch of workloads. Do we have any warning for this anywhere?

Reply:
We have a couple runbooks that detect HPP degradation and we have a warning about storage pools: "The storage pools must not be in the same partition as the operating system. Otherwise, the operating system partition might become filled to capacity, which will impact performance or cause the node to become unstable or unusable."

However, that is the extent of it. What kind of warning do you suggest?

Reply:
@aglitke please advise reasonable HPP caution message.